### PR TITLE
Move slots from DataGeneration to NucleotideSequencing

### PR DESCRIPTION
**Pull request by bmeluch**
See https://github.com/microbiomedata/nmdc-schema/issues/1760. 

Moved sequencing-specific slots from DataGeneration to NucleotideSequencing so that MassSpectrometry would stop inheriting irrelevant slots.

The valid example files for NucleotideSequencing are still valid. Added an invalid example file of a MassSpectrometry object which has a NucleotideSequencing slot.

### CODE REVIEW

**Review by mslarae13 — COMMENTED** [review]
I think this is goood. Alicia should definitely review too!

@bmeluch can you add valid examples?

File: src/data/invalid/MassSpectrometry-incl-nucleotideseq-slots.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+id: nmdc:dgms-99-9n9n9n
+add_date: 30-OCT-14 12.00.00.000000000 AM
+mod_date: 22-MAY-20 06.13.12.927000000 PM
+name: Thawing permafrost microbial communities from the Arctic, studying carbon transformations - Permafrost 712P3D
+processing_institution: JGI
+type: nmdc:MassSpectrometry
+alternative_identifiers:
```
Comment (aclum) [inline src/data/invalid/MassSpectrometry-incl-nucleotideseq-slots.yaml:7]: Invalid data files should only have one invalid slot at a time. This has a value for alternative_identifiers that doesn't make sense and has both target_gene and ncbi_project_name.

Comment (aclum) [reply src/data/invalid/MassSpectrometry-incl-nucleotideseq-slots.yaml:7]: The rest of the PR looks good to me.

Comment (bmeluch) [reply src/data/invalid/MassSpectrometry-incl-nucleotideseq-slots.yaml]: Fixed, thanks Alicia!

**Comment by aclum** [comment]
@bmeluch there are conflicts now, perhaps from @turbomam backmerge from nmdc-schema? can you them?

**Comment by bmeluch** [comment]
@aclum should be fixed now, thanks for reviewing!

**Review by aclum — APPROVED** [review]

**Pull request merged**